File: src/main/java/com/tartner/DanceHoursSpringBootApplication.java
package com.tartner;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackageClasses = DanceHoursSpringBootApplication.class)
public class DanceHoursSpringBootApplication {

	public static void main(String[] args) {
		SpringApplication.run(DanceHoursSpringBootApplication.class, args);
	}
}

File: src/main/java/com/tartner/DanceHoursSpringBootApplication.kt
package com.tartner

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication(scanBasePackageClasses = arrayOf(DanceHoursSpringBootApplication::class))
object DanceHoursSpringBootApplication {

    @JvmStatic fun main(args : Array<String>) {
        SpringApplication.run(DanceHoursSpringBootApplication::class.java, *args)
    }
}
